tests/security/audit/autrace.pm

```diff
@@ -0,0 +1,58 @@
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-Later
+#
+# Summary: Verify the "autrace" utility traces individual processes in a fashion similar to strace. The output of autrace is logged to the audit log.
+# Maintainer: llzhao <llzhao@suse.com>
+# Tags: poo#81772
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub get_pid {
+    # Call this fuction to extract pid from autrace output in command line
+    script_run("tail -1 /tmp/out > /tmp/backup");
+    script_run("awk -F\\' '{print \$2}' /tmp/backup > /tmp/out");
+    my $pid = script_output('cat /tmp/out | cut -d " " -f 4');
+    return ($pid);
+}
+
+sub run {
+    my $audit_log = '/var/log/audit/audit.log';
+
+    select_console 'root-console';
+
+    # Install related packages
+    zypper_call('in audit libaudit1');
```

Comment:
I didn't see any steps to start or check auditd services, does it start automatic after installing the pkgs above?

Reply:
I think it is automatically started after the downloading, but I just added a systemctl start to make sure.